prog/download_boot_image.rb

```diff
@@ -44,8 +44,16 @@ def url
     @url ||=
       if frame["custom_url"]
         frame["custom_url"]
-      elsif download_from_blob_storage?
-        blob_storage_client.get_presigned_url("GET", Config.ubicloud_images_bucket_name, "#{image_name}-#{vm_host.arch}-#{version}.raw", 60 * 60).to_s
+      elsif download_from_blob_storage? || Config.e2e_test?
```

Comment:
Should we download from the blob storage in production to make production and E2E environments more similar? I prefer to use the same file for both production and E2E.
maybe
```suggestion
      elsif download_from_blob_storage? || !Config.development?
```

Reply:
That is my intention as well, but I want to use this for E2E tests for a while (~1 week) before using it for prod.